src/views/forms/AccountManagementForms/SourceSelectMenu.tsx

```diff
@@ -85,10 +85,14 @@ export const SourceSelectMenu = ({
         </Styled.Text>
       ),
     }))
-    .filter(
-      (chain) =>
-        type === TransferType.deposit || !!cctpTokensByChainId[chain.value] || !CCTPWithdrawalOnly
-    )
+    .filter((chain) => {
```

Comment:
nit would this be equivalent to: `if (type === TransferType.deposit && CCTPDepositOnly) || (type === TransferType.widthdrawal && CCTPWithdrawalOnly) return  !!cctpTokensByChainId[chain.value] else true`

Reply:
is that easier for you to understand? i [just tried to write it in the way that felt the clearest for me to read](https://github.com/dydxprotocol/v4-web/pull/540/files#r1601887869) - if it's easier for other people to understand it that way i'll rewrite it!

Reply:
eh I don't wanna block on this nit; also don't think it's a big deal, up to you!

Reply:
synced offline and updated to a mix of both @moo-onthelawn's suggestion and my original!